lxd/storage/drivers/driver_zfs_utils.go

```diff
@@ -194,6 +194,40 @@ func (d *zfs) getDatasets(dataset string, types string) ([]string, error) {
 	return children, nil
 }
 
+func (d *zfs) filterRedundantOptions(dataset string, options ...string) ([]string, error) {
+	keys := []string{}
```

Comment:
You can just do `var keys, values []string` here as there's no need to initialise them, append will do this for you.